must-call-checker/src/main/java/org/checkerframework/checker/mustcall/MustCallAnnotatedTypeFactory.java

```diff
@@ -204,7 +204,7 @@ private void changeNonOwningParametersTypes(
   public AnnotatedTypeMirror fromElement(Element elt) {
     AnnotatedTypeMirror type = super.fromElement(elt);
     // Support @InheritableMustCall meaning @MustCall on all class declaration elements.
-    if (ElementUtils.isClassElement(elt)) {
+    if (ElementUtils.isTypeElement(elt)) {
```

Comment:
Let's not include this unrelated change. Probably @kelloggm has fixed on his branch